sonar-project.properties

```diff
@@ -1,2 +1,4 @@
 sonar.projectKey=one-day-intern_odi-assessee-fe
 sonar.organization=one-day-intern
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.exclusions= **/index.ts, **/cypress/**, **/*.d.ts, **/components/shared/**/svg/**, **/*.config.js, **/*.config.ts
```

Comment:
Please ensure that the excluded files will not contain any functionality related code.

Reply:
Okay Jo my bad, I accidentally added the wrong file to the sonarcloud exclusions